dotcom-rendering/src/components/Branding.importable.tsx

```diff
@@ -11,6 +11,37 @@ const brandingStyle = css`
 	padding-bottom: 10px;
 `;
 
+const brandingAdvertisingPartnerStyle = css`
+	margin: 4px 0 20px;
+	padding: 4px;
+	border: 1px solid ${palette('--branding-border')};
+	width: fit-content;
+
+	${from.desktop} {
+		padding: 8px;
+		width: 220px;
+	}
+	${from.leftCol} {
+		padding: 4px;
+		width: 140px;
+	}
+	${from.wide} {
+		padding: 8px;
+		width: 219px;
```

Comment:
Could you explain why this is 1px shorter?

Reply:
That is a good question! I have no idea why I thought it is not going to be aligned with the above `hr`(s) but I changed it now to `220px`